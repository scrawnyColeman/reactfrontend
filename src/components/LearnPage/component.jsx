import React from 'react';
import styled from 'styled-components';
import PopularCourses from '../PopularCourses/component';
import data from '../../data/courses.js';
import CoursesSearch from '../CoursesSearch/component';


const StyledItemWrapper = styled.div`
    display: grid;
    grid-template-columns: 4fr 3fr;
    grid-gap: 32px;
    margin: 20vh 20px 0 20px;
`;

const LearnPage = () => (
    <StyledItemWrapper>
        <CoursesSearch courses={data.courses} />
        <PopularCourses courses={data.courses} />
    </StyledItemWrapper>
);

export default LearnPage;