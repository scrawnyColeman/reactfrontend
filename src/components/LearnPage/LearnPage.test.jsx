import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-styled-components';
import React from 'react';
import LearnPage from './component';

Enzyme.configure({ adapter: new Adapter() });

describe('LearnPage component', () => {
    test('component matches snapshot', () => {
        const wrapper = shallow(<LearnPage />);
        expect(wrapper).toMatchSnapshot();
    });
});
