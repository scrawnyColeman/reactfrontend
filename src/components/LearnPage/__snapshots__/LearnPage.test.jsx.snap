// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LearnPage component component matches snapshot 1`] = `
<styled.div>
  <CoursesSearch
    courses={Array []}
    isLoading={false}
  />
  <PopularCourses
    courses={Array []}
    isLoading={false}
  />
</styled.div>
`;
